src/rgw/rgw_admin.cc

```diff
@@ -60,6 +60,7 @@ extern "C" {
 #include "rgw_bucket_sync.h"
 #include "rgw_sync_checkpoint.h"
 #include "rgw_lua.h"
+#include "rgw_sal_motr.h"
```

Comment:
better avoid including motr_sal layer header file here

Reply:
As discussed, we need this to evoke `MotrStore` specific function

Reply:
Use build time flag to cover this include statement

Reply:
Done